docs/installation/introduction.rst

```diff
@@ -119,32 +247,59 @@ other ranges during the
 :ref:`Bootstrap configuration <Bootstrap Configuration>`.
 
 
-.. _installation-intro-install-plan:
-
-Installation Plan
-^^^^^^^^^^^^^^^^^
-In this guide, the depicted installation procedure is for a medium-sized
-cluster, using three control plane nodes and two worker nodes. Refer to
-the :doc:`/installation/index` for extensive explanations of possible
-cluster architectures.
-
-.. note::
-
-   This image depicts the architecture deployed with this installation guide.
-
-   .. image:: img/architecture.png
-      :width: 100%
-
-   .. todo::
-
-      - describe architecture schema, include legend
-      - improve architecture explanation and presentation
-
-The installation process can be broken down into the following steps:
-
-#. :doc:`Setup <./setup>` of the environment (with requirements and example
-   OpenStack deployment)
-#. :doc:`Deployment <./bootstrap>` of the :term:`Bootstrap node`
-#. :doc:`Expansion <./expansion>` of the cluster from the Bootstrap node
-
-.. todo:: Include a link to example Solution deployment?
+Additional Notes
+^^^^^^^^^^^^^^^^
+
+.. _installation-intro-sizing:
+
+Sizing
+""""""
+Defining an appropriate sizing for the machines in a MetalK8s cluster strongly
+depends on the selected architecture and the expected future variations to
+this architecture. Refer to the documentation of the applications planned to
+run in the deployed cluster before completing the sizing, as their needs will
+compete with the cluster's.
+
+Each :ref:`role<node-roles>`, describing a group of services, requires a
+certain amount of resources for it to run properly. If multiple roles are used
+on a single Node, these requirements add up:
+
+- Bootstrap services, including Salt Master and its API, package repositories,
+  and container registries, need a minimum of 1 CPU core and 2 GB of RAM
+- Control plane services, including Kubernetes API and its backing ``etcd``
+  database, require at least 0.5 CPU core and 1.5 GB of RAM
+- Infra services, running Prometheus and Alertmanager among others, need 0.5
```

Comment:
Shouldn't we add some sizing for any node in the cluster? even if no bootstrap, control plane or infra services is running on it?

Reply:
maybe a little table where you see type of services, associated role and corresponding sizing (cpu, ram and storage) would be even more clear

Reply:
> Shouldn't we add some sizing for any node in the cluster?

The whole point is that we **cannot** know what sizing is required to run "any Node". That's why we explicitly state that users should _Refer to the documentation of the applications planned to run in the deployed cluster before completing the sizing, as their needs will compete with the cluster's_

> maybe a little table [...] would be even more clear

Will do.

Reply:
Obviously the sizing will depend on what application is running on it but the sizing due to MetalK8s services running on it (i.e. node exporter, kubelet, salt minion I guess) is either negligible either we could / should provide an estimation for it so that it is taken in account when building the global sizing of the cluster.

Reply:
I think they are negligible compared to the actual workloads you'll want to deploy on top of MetalK8s, but I can try to provide some sizing info.

@xaf-scality also suggested giving capacity required for the etcd + prometheus + alertmanager partitions, and add a recommendation for using a dedicated /var partition of 100+ GB